javascript/javascript1/week3/distance.js

```diff
@@ -0,0 +1,16 @@
+//When will we be there??
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function calTime(travelInformation)
+  {
+      let time = (travelInformation.destinationDistance)/(travelInformation.speed);
+      let hour = Math.trunc(time);
```

Comment:
Use `const` but also consider the following alternative:
```js
const hour = Math.trunc(travelInformation.destinationDistance / travelInformation.speed)
```
:point_up: the advantage is one less variable, the disadvantage might be you finding it harder to read... I just wanted to show the possibility.

Reply:
done